Assets/2_MainScene/S2_Scripts/S2_EnemyScripts/IKillable.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
```

Comment:
New scripts should be commented with name/date/last modified ~See tops of established scripts